manifests/v/v2raya/win/1.5.6.2/v2raya.win.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: v2raya.win
+PackageVersion: 1.5.6.2
+PackageLocale: en-US
+Publisher: v2rayA Developers

```

Comment:
```suggestion
Publisher: The v2rayA developer community
```

Reply:
Thanks for your advise, This change is acceptable and good

Reply:
cc @MarksonHon: Would you be able to commit the changes on your end?